packages/manager/modules/pci/src/projects/new/payment/components/add/index.html

```diff
@@ -0,0 +1,48 @@
+<div class="pci-project-new-payment-method-add">
+    <div
+        class="row"
+        data-ng-repeat="authorizedPaymentMethods in $ctrl.authorizedPaymentMethods.chunks track by $index"
+    >
+        <div
+            class="col-md-6 mb-3"
+            data-ng-repeat="authorizedPaymentMethod in authorizedPaymentMethods track by $index"
+        >
+            <oui-radio
+                class="oui-radio_btn"
+                data-name="paymentType"
+                data-thumbnail
+                data-variant="light"
+                data-model="$ctrl.model.paymentMethod"
+                data-value="authorizedPaymentMethod"
+                data-on-change="$ctrl.onPaymentTypeRadioChange()"
+                data-disabled="$ctrl.globalLoading.finalize"
+            >
+                <span
+                    data-ng-bind="$ctrl.getPaymentMethodTypeText(authorizedPaymentMethod)"
+                ></span>
+            </oui-radio>
+        </div>
+    </div>
+
+    <div data-ng-if="$ctrl.model.paymentMethod.paymentType !== 'CREDIT'">
+        <p data-translate="pci_project_new_payment_method_add_explain"></p>
+
+        <oui-message
+            data-type="warning"
+            data-ng-if="$ctrl.model.paymentMethod.paymentType === 'BANK_ACCOUNT'"
+        >
+            <span
+                data-translate="pci_project_new_payment_method_add_warning_bank_account"
+            ></span>
+        </oui-message>
+
+        <p>
+            <small
```

Comment:
`small` has a semantic meaning: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/small

is it he case here ?